### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#2148] [Q&A] [answered]
Hi everyone.

I want to help, but I want to know if this project is active. *Is this project active?*

#### ideas
- [Min Browser and Keepass](https://github.com/minbrowser/min/discussions/2088)
- [option for disable header menu bar in browser Min](https://github.com/minbrowser/min/discussions/2091)
- [min browser and findTo ](https://github.com/minbrowser/min/discussions/2137)

**Note:** I'm thinking of implementing these ideas, but I need feedback!

*What do you all think of the idea?*

**Comment by PalmerAL** [comment, marked as answer]
I've reduced the time I spend working on this, but if you make a PR for something, I'm happy to review it. There are still a lot of people that actively use this project (and I intend to continue releasing updates), so I'm sure it'd be appreciated.

**Reply by ghost** [reply]
Hi PalmerAL.

> I've reduced the time I spend working on this, but if you make a PR for something, I'm happy to review it. There are still a lot of people that actively use this project (and I intend to continue releasing updates), so I'm sure it'd be appreciated.

thank you for feedback again.
